packages/blade/src/_helpers/testing/assertAccessibe.web.ts

```diff
@@ -0,0 +1,10 @@
+import { axe } from 'jest-axe';
+import type { JestAxeConfigureOptions } from 'jest-axe';
+
+export default async function assertAccessible(
+  container: string | Element,
+  axeOptions?: JestAxeConfigureOptions,
+): Promise<void> {
+  const results = await axe(container, axeOptions);
+  expect(results).toHaveNoViolations();
```

Comment:
This seems too vague to understand. What is the scope of this check? Like what all would it catch and what would it not?